ntoskrnl/include/internal/se.h

```diff
@@ -71,6 +61,14 @@ typedef struct _TOKEN_AUDIT_POLICY_INFORMATION
 #define TOKEN_DUPLICATE_METHOD 0xDUL
 #define TOKEN_FILTER_METHOD    0xFUL
 
+//
+// Access Rights Types Array
+//
+#define ACCESS_RIGHTS_REMAINING_RIGHTS_ARRAY  0
```

Comment:
Names are quite long, maybe use "ACCESS_RIGHTS_REMAINING" etc. (no need to say "RIGHTS_ARRAY"